tests/test_cli_sync.py

```diff
@@ -384,4 +384,10 @@ def test_config_option(run, runner, make_config_file):
     out = runner.invoke(cli, ["--config", config_file.as_posix()])
 
     assert out.exit_code == 1
-    assert "Would install:" in out.stdout
+    dry_run_message = "Would install:"
+    assert dry_run_message in out.stdout
+
+    out = runner.invoke(cli, ["--no-config", "--config", config_file.as_posix()])
```

Comment:
This should be a separate test with its own name or at least a parametrized case..

Reply:
I prefer triangulation technique here instead.

Reply:
I don't see how it's applicable in this case. You're clearly testing two separate behaviors that must appear as separate entries in the test report.
The only way of having two things reported from the same test function is to use `pytest-subtest`. But it still feels wrong since you're not checking two properties of the same behavior outcome.

Reply:
Also, for property-based testing, we should really integrate Hypothesis..

Reply:
Here you go 9a474fbb07913dbbf3c14b276294bdf91d8b8f45

Reply:
Thanks!